packages/examples/cvat/exchange-oracle/src/handlers/job_creation.py

```diff
@@ -550,6 +550,12 @@ def build(self):
         return super().build()
 
 
+class PolygonTaskBuilder(SimpleTaskBuilder):
+    def _setup_quality_settings(self, task_id: int, **overrides) -> None:
+        values = {"iou_threshold": Config.core_config.polygons_iou_threshold, **overrides}
```

Comment:
```suggestion
        values = {"iou_threshold": Config.core_config.polygons_iou_threshold}
        values.update(**overrides)
```
The code can fail if there is `iou_threshold` in `overrides`.

Reply:
Python allows duplicate keys in dict literal, the last key will remain so result would be identical in this case.

```python
print({"foo": 1, "foo": 2})  # {"foo": 2}
print({"foo": 1, **{"foo": 2}})  # {"foo": 2}
```

You might have confused this with function arguments expansion in function calls (usually `**kwargs`), in this case, python indeed doesn't allow multiple arguments to be present.